scenes/game/remote_game.gd

```diff
@@ -545,3 +547,6 @@ func do_match_result(player_clock_remaining, opponent_clock_remaining):
 	}
 	_submit_game_message(action_message)
 	return true
+	
+func trigger_timeout(event_player : Enums.PlayerId, reason : Enums.GameOverReason):
+	local_game.do_timeout(event_player, reason)
```

Comment:
This is not correct.
Please see every other do_* and process_* function in this file.
The idea is that when your clock runs out, you call the do_*, which sends the message to the game server.
When the gameroom repeats the message back to both clients, you get the message via the process_* function, and then you call the actual local_game.do_* function.